kafka-message-signing/src/main/java/com/alliander/osgp/kafka/message/signing/UncheckedSecurityException.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2022 Alliander N.V.
```

Comment:
Dit is nog het oude formaat copyright (en oud jaartal ;) )

Reply:
En ik denk dat hier dan ook de open source license op komt te staan

Reply:
check